index.js

```diff
@@ -156,8 +156,18 @@ Plugin.prototype.readFile = function(file, callback) {
 			middleware.fileSystem.readFile("/_karma_webpack_/" + file.replace(/\\/g, "/"), callback);
 		}
 	}
-	if(!this.waiting)
-		doRead();
+	if (!this.waiting) {
```

Comment:
I would have assumed so, but from testing on my local machine (Windows 7 x64), `this.waiting` was evaluating to false whenever a new file was added that matched the glob during a watch.

I really don't understand the internals that are going on here. I simply used the logic that I would have expected to run (in the `this.waiting == true` branch) in the case that we observed the specific error that was getting thrown. It's entirely possible there's a simpler / more elegant way to resolve this issue. If you've got any suggestions or improvements let me know and I'd be happy to update things.